ui/page/show/view.jsx

```diff
@@ -142,38 +138,26 @@ function ShowPage(props: Props) {
         {!isResolvingUri && isSubscribed && claim === null && <AbandonedChannelPreview uri={uri} type={'large'} />}
       </Page>
     );
-  } else if (claim.name.length && claim.name[0] === '@') {
+  } else if (claim && claim.name.length && claim.name[0] === '@') {
     innerContent = <ChannelPage uri={uri} location={location} />;
+  } else if (isBlacklisted && !claimIsMine) {
+    innerContent = (
+      <Page>
+        <Card
+          title={uri}
+          subtitle={__(
+            'In response to a complaint we received under the US Digital Millennium Copyright Act, we have blocked access to this content from our applications.'
```

Comment:
We hvae a few cases:
1) it literally says DMCA in the channel name -> show DMCA text, but say "hub" did it - not "we" did it.
2) the channel that blocks it has a description -> some way to show channel and description
3) the channel that blocks it has no description. -> some way to show channel and generic text

I think this may need some way to display the channel that blocked (reposted) it, to resolve the channel that blocked it (possibly a claim preview component?) and display the description of the channel that blacked it.